### DISCUSSION TITLE
Amazon 的科技監控

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#427] [General]
這在亞洲可能沒人有意見... 我就奴

https://technews.tw/2022/05/27/amazon-installs-ai-cameras-to-monitor-its-delivery-drivers/

**Comment by luo910227** [comment]
如果我是司機應該會想辭職吧，這樣送貨壓力太大了，如果本來就是負責人的人還要被監控會很不甘
